card/spec/lib/card/set/trait_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe Card::Set::Trait do
+  class Card
+    module Set
+      module Type
+        module Phrase
+          extend Card::Set
+          card_accessor :write, type: :pointer
+          card_accessor :read, type_id: Card::PointerID
```

Comment:
ah, ok.  If you don't understand why :type doesn't work (I don't), let's at least make it fail loudly.